cddl/update-record.cddl

```diff
@@ -0,0 +1,12 @@
+update-triple-record = [
+&(subj: 0) => $subject-type-choice
+& (mval: 1) => [+measurement-values-map] ; object of the update
```

Comment:
Why call it `mval` if it means "object"?

Reply:
because that is triple speak? If not, that's a good question.